source/request.js

```diff
@@ -181,6 +181,14 @@ define(['airports'], function (airports) {
     return this.getCloudFlareLocationCode();
   };
 
+  Request.prototype.getCloudFlareTrace = function () {
```

Comment:
A global [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) object is available in the extension process. This function can be simplified to use it, such as the following:

```javascript
Request.prototype.getCloudFlareTrace = function () {
  var traceURL = new URL(this.details.url);
  traceURL.pathname = '/cdn-cgi/trace';
  return traceURL.toString();
};
```